lib/Avatar/index.js

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import cx from 'classnames';
 
 const Avatar = (props) => {
-  const styleClass = props.fadedOut ? 'is-faded' : '';
+  const styleClass = cx({
+    'is-faded': props.fadedOut,
+    'is-assigned': props.isAssigned,
```

Comment:
It's ideal not to tie names in a UI library too much with business logic. Lets rename this to something a little more generic/abstract. For example `avatar--highlight` could map to `props.isHighlighted`.